chatmaild/src/chatmaild/tests/test_doveauth.py

```diff
@@ -120,28 +121,31 @@ def test_handle_dovecot_request(db, example_config):
 
 
 def test_handle_dovecot_protocol_hello_is_skipped(db, example_config, caplog):
+    dictproxy = AuthDictProxy(db=db, config=example_config)
     rfile = io.BytesIO(b"H3\t2\t0\t\tauth\n")
     wfile = io.BytesIO()
-    handle_dovecot_protocol(rfile, wfile, db, example_config)
+    dictproxy.loop_forever(rfile, wfile)
     assert wfile.getvalue() == b""
     assert not caplog.messages
 
 
 def test_handle_dovecot_protocol(db, example_config):
+    dictproxy = AuthDictProxy(db=db, config=example_config)
     rfile = io.BytesIO(
         b"H3\t2\t0\t\tauth\nLshared/userdb/foobar@chat.example.org\tfoobar@chat.example.org\n"
     )
     wfile = io.BytesIO()
-    handle_dovecot_protocol(rfile, wfile, db, example_config)
+    dictproxy.loop_forever(rfile, wfile)
     assert wfile.getvalue() == b"N\n"
```

Comment:
So the `protocol` test tests failing authentication and the `request` test tests for working authentication? Maybe we could rename them so it is more obvious on first glance what they do.

Reply:
renamed test